scripts/setup-first-time.sh

```
@@ -0,0 +1,195 @@
+    echo "Error: Failed to download Docker templates"
+    exit 1
+fi
+
+# Move the Docker files to our installation directory
+mv compose/* .
+mv compose/.gitignore .
+mv compose/.vscode .
+rm -rf compose .git
+
+# Go back to root directory
+cd $ROOT_DIR
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)